### PR TITLE
Ultrasonic obstacle avoidance

### PR DESCRIPTION
**Pull request by pontuslaestadius**
Tested on the vehicle,

Can be tested by just running docker-compose up as the image is already loaded there.
Will print three different distances of obstacles.

Does not include libcluon as it has a Debian issue.

### CODE REVIEW

File: .CHAIN_HELP [hunk 1]
```diff
@@ -12,3 +12,4 @@ It is prefered to delete your build directory before running.
 -t	  --transfer	  saves the docker image and sends it to the device over ssh followed by execution.
 -ba	  --build-arm	  builds a docker image using the .armhf Dockerfile in current directory.
 -b        --build         builds a docker image using the default Dockerfile in current directory.
+-new-microservice	  needs 1 argument, can not be combined with other arguments. Creates a new microservice in the provided directory, using files in current directory.
```
Comment (kosara-g) [inline .CHAIN_HELP:15]: when you write "provided directory", what would that be?

File: docker-compose.yml [hunk 2]
```diff
@@ -20,10 +20,33 @@ services:
...
+    #    image: chalmersrevere/opendlv-signal-viewer-armhf:v0.0.4
+    #    network_mode: "host"
+    #    restart: always
+    #    privileged: true
+    #    expose:
+    #    - "8080"
+    #    depends_on:
+    #    - odsupercomponent
+    #    command: "/opt/signal-viewer --cid=111"
+
```
Comment (kosara-g) [inline docker-compose.yml]: is this tested?

File: .CHAIN_HELP (hunk 1 above)
Comment (pontuslaestadius) [reply .CHAIN_HELP:15]: sh chain.sh -new-microservice newdirectory
Will make a new directory inside the microservices directory with some basic files copied over.

File: docker-compose.yml (hunk 2 above)
Comment (pontuslaestadius) [reply docker-compose.yml]: Removed it.

**Review by kosara-g — APPROVED** [review]
Cluon issue should be fine

**Pull request merged**